### ISSUE TITLE
Nodos Vecinos

### ISSUE THREAD

**Issue by fringlesinthestreet** [IIC2133-PUC/2017-2#96]
Hola :)

¿En la tarea son vecinos 2 nodos si se conectan de forma diagonal? Imaginen una imagen con 4 colores que en el medio se ve así:
1  1  2 2
1  1  2 2
3 3 4 4
3 3 4 4

¿Los nodos 1 son vecinos con 2, 3 y 4? o sólo 2 y 3.

**Comment by alopez7** [comment]
No, solo son vecinos si colindan arriba, abajo, izquierda o derecha

**Comment by fringlesinthestreet** [comment]
gracias :)
